benchmark/torch/RL4LMs/requirements.txt

```diff
@@ -0,0 +1,12 @@
+parl==2.1.1
+datasets==2.10.1
+PyYAML==6.0
```

Comment:
remove this third-party dependency as we use the python dictionary as the config.